regression/scripts/Vehicles/Boeing_BWB_450.py

```diff
@@ -30,8 +30,7 @@ def vehicle_setup():
     # ------------------------------------------------------------------    
 
     vehicle = SUAVE.Vehicle()
-    vehicle.tag = 'Boeing_BWB_450'    
-
+    vehicle.tag = 'Boeing_BWB_450'
```

Comment:
why is this file changed?

Reply:
Not sure why the changed but I pulled in the develop branch's script